bolero-libfuzzer/fix-fork.patch

```diff
@@ -0,0 +1,400 @@
+diff -ruN libfuzzer/FuzzerCommand.h libfuzzer-new/FuzzerCommand.h
```

Comment:
This patch seems bigger than it needs to be. Can we isolate the change to this single function?

Reply:
We could, but it would mean that literally every run would need to go through the libtest harness. I thought every runs did, but then saw in the bolero source code that there’s a BOLERO_LIBTEST_HARNESS environment variable, so I considered it’s not actually the case?

(Also, I thought maybe libfuzzer also needs to be able to run executables without going through the libtest harness, eg. in the dataflow things, but I have literally no idea how those works so I wrote the patch only so it doesn’t newly break stuff, at the expense of maybe not fixing them if they were already broken)

Reply:
That said if bolero fuzzers always run through the libtest harness I could at least slightly simplify the patch :)

Reply:
I've opened #176 as a more minimal change. The `BOLERO_LIBTEST_HARNESS` environment variable is just to detect if we're using libtest or not. If we're not, it would look something like: 

```rust
fn main() {
    bolero::check!().for_each(|_| { });
}
```

Which means we still go through bolero's harness. But it'll only execute a single harness in this mode.